elm.json

```diff
@@ -1,57 +1,58 @@
 {
```

Comment:
I think you accidentally prettifyed `elm.json`, the elm-compiler does not like handmade changes to it...